From 939100f1296c1be4ba672c3d5aed171a661407c5 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Thu, 9 Aug 2018 12:42:46 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3308-evb: switch PCM master for AP
 side

We may need to support a variety of different Bluetooth modules
which are from different vendors. In order to develop efficiently,
we need to uniformly set the PCM of the AP to master mode, and
the Bluetooth side as the slave.

Change-Id: I92e76a9e394eea7b3f700bada5ad97ae92a67c69
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 2 --
 arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi | 2 --
 2 files changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 059df0fb8d2e..d10f5442550b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -120,8 +120,6 @@
 		rockchip,codec = <&dummy_codec>;
 		rockchip,format = "dsp_b";
 		rockchip,bitclock-inversion = <0>;
-		rockchip,bitclock-master = <&dummy_codec>;
-		rockchip,frame-master = <&dummy_codec>;
 		rockchip,wait-card-locked = <0>;
 	};
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
index 32f4d225edd4..68483830fb53 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
@@ -105,8 +105,6 @@
 		rockchip,codec = <&dummy_codec>;
 		rockchip,format = "dsp_b";
 		rockchip,bitclock-inversion = <0>;
-		rockchip,bitclock-master = <&dummy_codec>;
-		rockchip,frame-master = <&dummy_codec>;
 		rockchip,wait-card-locked = <0>;
 	};
 
-- 
2.35.3

